src/main/java/core/basesyntax/strategy/impl/BirthdayDiscountService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class BirthdayDiscountService implements DiscountService {
+    @Override
+    public double getDiscount() {
+        return (double) 33;
```

Comment:
```suggestion
        return 33;
```

we dont need this cast (same in other classes)